src/schemas/enums/GitTokenAssociatedServiceType.yaml

```diff
@@ -1,5 +1,6 @@
 type: string
 enum:
   - APPLICATION
-  - JOB
```

Comment:
Will it break the CLI and Terraform to remove JOB?

Reply:
I don't think as I don't see any related calls inside cli or terraform